Breeze.UI/src/app/setup/recover/recover.component.ts

```diff
@@ -144,14 +151,15 @@ export class RecoverComponent implements OnInit {
   }
 
   private recoverStratisWallet(recoverWallet: WalletRecovery, bitcoinErrorMessage: string){
-    let stratisErrorMessage = "";
+    let stratisErrorMessage = '';
     this.apiService
       .recoverStratisWallet(recoverWallet)
       .subscribe(
         response => {
           if (response.status >= 200 && response.status < 400) {
-            let body = "Your wallet has been recovered. \nYou will be redirected to the decryption page.";
-            this.genericModalService.openModal("Wallet Recovered", body);
+            const body =
```

Comment:
I'd argue this is harder to read on two lines

Reply:
It was autoformatted by tslint. I do like it as on the smaller laptop (like mine) it is easier to read without having to scroll